test/utils/wrap-display-name.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { wrapDisplayName } from '../../src/utils'
+
+describe('Wrap Display Name', () => {
+  test('wraps the display name of a React component with the name of an HoC', () => {
```

Comment:
A little pedantic but I think omitting "display" here will make this test more clearly stand out in comparison to the next test case